src/test/clip.t.sol

```diff
@@ -169,55 +145,44 @@ contract DutchClipperTest is DSTest {
 
         me = address(this);
 
-        gov = new DSToken('GOV');
-        gov.mint(100 ether);
-
-        vat = new TestVat();
+        vat = new Vat();
         vat = vat;
```

Comment:
Am I missing something or can this line be removed?

Reply:
Yeah, hadn't realized it was there.